dcspy/AH-64D_BLK_II.yaml

```diff
@@ -1,8 +1,8 @@
-DOWN: PLT_EUFD_PRESET CUSTOM PLT_EUFD_PRESET 0|PLT_EUFD_PRESET 1|
-FOUR: PLT_EUFD_ENT CUSTOM PLT_EUFD_ENT 0|PLT_EUFD_ENT 1|
-LEFT: PLT_EUFD_IDM CUSTOM PLT_EUFD_IDM 0|PLT_EUFD_IDM 1|
-ONE: PLT_EUFD_IDM CUSTOM PLT_EUFD_IDM 0|PLT_EUFD_IDM 1|
-RIGHT: PLT_EUFD_RTS CUSTOM PLT_EUFD_RTS 0|PLT_EUFD_RTS 1|
```

Comment:
This `PLT_EUFD_RTS` and the other `PLT_EUFD_RTS` (for TWO) aren't actually push buttons. The original test data was probably incorrect. The `PLT_EUFD_RTS` is a rocker switch with multiple positions so should actually be:

`RIGHT: PLT_EUFD_RTS CUSTOM PLT_EUFD_RTS 0|PLT_EUFD_RTS 1|`
`TWO: PLT_EUFD_RTS CUSTOM PLT_EUFD_RTS 2|PLT_EUFD_RTS 1|`

As per the description for that one: "switch position -- 0 = Down, 1 = Mid,  2 = Up"

Reply:
Yes, Anyway I need test Apache anyway.